### PR TITLE
[docs] update config-reference for 4.1.0 docs release

### PR DESCRIPTION
**Pull request by sarah11918**
## Changes

- Updates the wrong version in a `<Since />` component showing the version added
- Adds docs (we'll see whether I got the indentation correct!) for the new `limitInputPixels` config option that was only shown in an example, not documented.

## Testing

no tests, only docs

## Docs

Docs update

### CODE REVIEW

**Review by TheOtterlord — APPROVED** [review]
Indentation looks great!

File: packages/astro/src/@types/astro.ts [hunk 1]
```diff
@@ -1112,6 +1112,20 @@ export interface AstroUserConfig {
...
+			 * @type {boolean}
+			 * @default `true`
+			 * @version 4.1.0
+			 * @description
+			 *
+		   * Whether or not to limit the size of images that the Sharp integration will process.
+		   *
+			 * Set `false` to bypass the default image size limit for the Sharp integration and process large images.
+			 */
+			prefixDefaultLocale: boolean;
```
Comment (Princesseuh) [inline packages/astro/src/@types/astro.ts:1128]:
```suggestion
```

File: packages/astro/src/@types/astro.ts [hunk 2]
```diff
@@ -1112,6 +1112,19 @@ export interface AstroUserConfig {
...
+			/**
+			 * @docs
+			 * @name image.service.config.limitInputPixels
+			 * @kind h4
+			 * @type {boolean}
+			 * @default `true`
+			 * @version 4.1.0
+			 * @description
+			 *
+		   * Whether or not to limit the size of images that the Sharp integration will process.
```
Comment (Princesseuh) [inline packages/astro/src/@types/astro.ts:1124]:
```suggestion
		   * Whether or not to limit the size of images that the Sharp image service will process.
```

File: packages/astro/src/@types/astro.ts [hunk 3]
```diff
@@ -1112,6 +1112,19 @@ export interface AstroUserConfig {
...
+		   *
+			 * Set `false` to bypass the default image size limit for the Sharp integration and process large images.
```
Comment (Princesseuh) [inline packages/astro/src/@types/astro.ts:1126]:
```suggestion
			 * Set `false` to bypass the default image size limit for the Sharp image service and process large images.
```

Wish we could've linked to Sharp's docs for this option, unfortunately there's no way to make a permalink to it...

**Pull request merged**